lib/db/index.js

```diff
@@ -220,7 +225,24 @@ Db.prototype.exec = function (query, options) {
 
   this.server.logger.debug('executing query against db ' + this.name + ': ' + query);
 
-  return this.send('command', data);
+  this.emit("beginQuery", {
```

Comment:
Because `data` also includes `sessionId`, `database` and `db`, I don't thing this API needs all those